src/main/java/rcms/utilities/daqexpert/reasoning/logic/failures/deadtime/BackpressureFromEventBuilding.java

```diff
@@ -0,0 +1,122 @@
+package rcms.utilities.daqexpert.reasoning.logic.failures.deadtime;
+
+import org.apache.log4j.Logger;
+import rcms.utilities.daqaggregator.data.BU;
+import rcms.utilities.daqaggregator.data.DAQ;
+import rcms.utilities.daqaggregator.data.FED;
+import rcms.utilities.daqaggregator.data.RU;
+import rcms.utilities.daqexpert.FailFastParameterReader;
+import rcms.utilities.daqexpert.Setting;
+import rcms.utilities.daqexpert.reasoning.base.action.SimpleAction;
+import rcms.utilities.daqexpert.reasoning.logic.basic.ExpectedRate;
+import rcms.utilities.daqexpert.reasoning.logic.basic.NoRateWhenExpected;
+import rcms.utilities.daqexpert.reasoning.logic.basic.Transition;
+import rcms.utilities.daqexpert.reasoning.logic.failures.KnownFailure;
+import rcms.utilities.daqexpert.reasoning.logic.basic.Parameterizable;
+
+import java.util.*;
+
+/**
+ * Logic module identifying the reason behind deadtime
+ */
+public class BackpressureFromEventBuilding extends KnownFailure implements Parameterizable {
+
+    private static final Logger logger = Logger.getLogger(BackpressureFromEventBuilding.class);
+
+    private static Integer fedBackpressureThreshold;
+    private Integer evmFewRequestsThreshold;
+
+    public BackpressureFromEventBuilding() {
+        this.name = "Backpressure from Event Builder";
+
+        this.description = "Backpressure from Event Building (i.e. from RU but not from BUs)";
+
+        this.action = new SimpleAction("Call Remi or Andre.");
```

Comment:
Call the DAQ on-call mentioning that we have backpressure from the event building.